requirements.txt

```diff
@@ -143,10 +145,10 @@ pydyf==0.7.0
     # via weasyprint
 pymupdf==1.22.5
     # via -r requirements.in
+pypdf==3.13.0
+    # via notifications-utils
 pypdf2==2.10.9
```

Comment:
is this going to cause big issues? should we migrate this to use pypdf [3]?

Reply:
especially since we did the migration to solve a security vuln warning right?

Reply:
yes 🙌  I rushed 😢